go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/ScaleFT/sshkeys
+
+require (
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/dchest/bcrypt_pbkdf v0.0.0-20150205184540-83f37f9c154a
+	github.com/pmezard/go-difflib v1.0.0 // indirect
+	github.com/stretchr/testify v1.2.2
+	golang.org/x/crypto v0.0.0-20181106171534-e4dc69e5b2fd
```

Comment:
i'd like to be able to keep these dependencies free-standing -- i'm not super familiar with go.mod patterns yet (we're still mostly not yet using it).  ideally I won't want this library holding up min versions with things like x/crypto

Reply:
Since golang.org/x/crypto do not have tag I'm not sure we can define the min supported version without specify the specific commit hash

Reply:
How about we skip the go.mod / vendor modules changes for this PR -- keep it focused on the new algo + testing changes to support testing it (i'm fine with the test package rename).

I'll do some poking on how to best support the go.mod going forward, and that should be an easy separate PR I think once I understand the choices.

Reply:
Ok, I'll revert changes related to go-mod and travis CI